### PR TITLE
Update ROADMAP.md

### PR DESCRIPTION
**Pull request by hellonadya**
- add 6.3 roadmap

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -5,6 +5,41 @@ This document provides the roadmap of the planned ONLYOFFICE Document Server cha
...
 We also reserve the right to change it when necessary.
 
+## Version 6.3
+
+### All editors
+
+* Dark theme
+* Adding/editing/deleting passwords in the online version
+* 150% scaling
+* Adding files to Favorites folder
```
Comment (ShockwaveNN) [inline ROADMAP.md:15]: I think should be note that this should be implemented on DMS side also

**Review by ShockwaveNN — APPROVED** [review]

**Comment by ShockwaveNN** [comment]
Also maybe old entres, like 6.2 should be deleted altogether from this doc
Because 6.2 for example contains "Forms tab" which was not included in final release

File: ROADMAP.md (hunk 1 above)
Comment (hellonadya) [reply ROADMAP.md:15]: for adding to favs? or for smt else too?

Comment (ShockwaveNN) [reply ROADMAP.md:15]: I think only favs

**Review by ShockwaveNN — CHANGES_REQUESTED** [review]

File: ROADMAP.md (hunk 1 above)
Comment (hellonadya) [reply ROADMAP.md]: added

**Review by ShockwaveNN — APPROVED** [review]

**Review by trofim24 — APPROVED** [review]

**Pull request merged**